.gitignore

```diff
@@ -44,4 +44,7 @@ Thumbs.db
 !apps/server/tests/.env
 
 # Auto-generated schemas
-*.gql
\ No newline at end of file
+*.gql
+
+# queries/mutation generation script
+generate.sh
```

Comment:
wrote simple script to automate generation of generated.tsx file - let me know if You'd like me to commit it

Reply:
def should be commited!

Reply:
under bin/front/generate_schema or smth like that